docs/pages/distribution/publishing-websites-webpack.mdx

```diff
@@ -0,0 +1,337 @@
+---
+title: Publish websites with Webpack
+description: Different ways to publish the Expo web app with third-party services.
+---
+
+import { APIBox } from '~/components/plugins/APIBox';
+import { Step } from '~/ui/components/Step';
+import { Tab, Tabs, TabsGroup } from '~/ui/components/Tabs';
+import { Table, TableHead, Row, HeaderCell } from '~/ui/components/Table';
+import { Terminal } from '~/ui/components/Snippet';
+import { YesIcon, NoIcon, PendingIcon } from '~/ui/components/DocIcons';
+
+> **warning** In SDK 50 and above, publishing with `webpack` is deprecated.
+
+A web app created using Expo can be served locally for testing out the production behavior. Once the testing phase checks out, you can choose from a variety of third-party services to host it.
+
+| Bundler   | Expo Router | API Routes | Description                                                                                |
+| --------- | ----------- | ---------- | ------------------------------------------------------------------------------------------ |
+| `webpack` | <NoIcon />  | <NoIcon /> | Outputs a Single Page Application (SPA) with a single **index.html** in the output folder. |
+
+## Create a build
+
+Creating a build of the project is the first step to publishing a web app. Whether you want to serve it locally or host it on a third-party service, you'll need to export all JavaScript and assets of a project. This is known as a static bundle. It can be exported by running the following command:
+
+Run the webpack export command to compile the project for web:
+
+<Terminal cmd={['$ npx expo export:web']} />
+
+The resulting project files are in the **web-build** directory. Any files inside the **web** directory are also copied to the **web-build** directory.
+
+If you make changes to your project, you'll need to rebuild it for production. Don't edit the **web-build** directory directly.
+
+> **Tip:** If you want to serve your site in a subfolder, add its path to your **package.json** as shown below:
+>
+> ```json package.json
+> {
+>   "homepage": "/path/to/subfolder"
+> }
+> ```
```

Comment:
````suggestion
> **Tip:** If you want to serve your site in a subfolder, add its path to your **package.json** as shown below:
>
> ```json package.json
> {
>   "homepage": "/path/to/subfolder"
> }
> ```
````

Let's remove the callout here. For consistency, I'd like us to avoid adding code snippets (other than inline ones)  in callouts. It also has broken styles.

![CleanShot 2023-12-19 at 02 06 23](https://github.com/expo/expo/assets/10234615/6494ae21-b62d-4763-aa9c-1c4a7fc08fa7)


Let's add this as a part of the main content. And since this is a "Tip", let's use **info** callout type here.